public/datasets/loneliness_interests.json

```diff
@@ -0,0 +1,181 @@
+{
+  "name": "loneliness_interests",
+  "card": {
+    "description": "A dataset of survey responses gathered by Slovakian students aged 18-21 at FSEV UK. Most responses are rankings on a scale from 1 to 5. For example, 'Rank from 1 (strongly disagree) to 5 (strongly agree): I often study or work even in my spare time'. This is a subset of the larger dataset that just focuses on personal interests",
```

Comment:
Should possibly be Slovak?  Not sure if [this](https://theculturetrip.com/europe/slovakia/articles/6-stereotypes-about-slovaks-that-simply-arent-true/) is authoritative.

Reply:
I think you're right. I looked at the original Kaggle dataset and they use this Slovak as an adjective, never Slovakian. This affects all of the student_interest datasets we have - great catch!